microservices/scraper-vietgate/app.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+from scraper import get_page_data
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def my_first_route():
```

Comment:
Quick fix - I would rename this route to something more specific

Reply:
Yep, that's on me - I was just directly running the scrapers on CLI